app.yaml

```diff
@@ -12,23 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-runtime: nodejs14
+runtime: nodejs12
 service: default
 instance_class: F4_1G
 
 # We fetch comments 500 at a time and send requests to score them all at once,
 # so we need a reasonable number of instances for this to not be slow.
 automatic_scaling:
-  max_instances: 1
-  min_instances: 0
```

Comment:
@kilemensi Updated, but I believe we should leave these `automatic_scaling` values as we had done, or should I just match them

Reply:
In this branch, match them exactly... Once we're ready to merge to main, we can decide which `app.yml` to keep @kelvinkipruto.